bedrock/newsletter/templates/newsletter/includes/newsletter-strings.json

```diff
@@ -110,12 +104,6 @@
     "description": "{{ ftl('newsletters-discover-the-latest')|striptags }}",
     "title": "{{ ftl('newsletters-firefox-os')|striptags }}"
   },
-  "marketplace-android": {
-    "title": "{{ ftl('newsletters-android')|striptags }}"
-  },
-  "marketplace-desktop": {
-    "title": "{{ ftl('newsletters-desktop')|striptags }}"
-  },
```

Comment:
**suggestion:** it looks like these two Fluent strings are no longer used anywhere. Suggest removing them if so?

Reply:
Removed the Fluent strings.